app/datatables/sample_datatable.rb

```diff
@@ -259,9 +259,9 @@ def sort_records(records)
             .select('"samples"."id"')
             .distinct
 
-          # grabs the ids that are not the previous one but are still of the same organization
+          # grabs the ids that are not the previous one but are still of the same team
           unassigned = Sample
-            .where('"samples"."organization_id" = ?', @organization.id)
+            .where('"samples"."team_id" = ?', @team.id)
```

Comment:
Hound is love, Hound is life :dog:

Probably better to move Sample into new line?